_posts/2023-11-21-trino-summit-2023-nears-lineup.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: "Trino Summit 2023 nears with an awesome lineup"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2023/lineup-blog-banner.png
+---
+
+As winter nears, the days may be getting shorter, but so is the wait until
+Trino Summit 2023! It'll be here before you know it on December 13th and 14th,
+and we've got a packed speaker lineup full of exciting talks that we're ready to
+share with the Trino community today. Read on for a preview of all the talks
+we're looking forward to, and if you're interested in attending, make sure to...
+
+<div class="card-deck spacer-30">
+    <a class="btn btn-pink" href="https://www.starburst.io/info/trinosummit2023/?utm_source=trino&utm_medium=website&utm_campaign=NORAM-FY24-Q4-EV-Trino-Summit-2023&utm_content=blog-lineup-announcement">
+        Register!
+    </a>
+</div>
+<div class="spacer-30"></div>
+
+<!--more-->
+
+So, who's going to be talking at Trino Summit? Here's a quick rundown of the
+talks coming in from various companies.
+
+## Starburst
+
+As always, our keynote will come from Martin Traverso, Trino co-founder and
+co-CTO at Starburst. He'll be giving a project update on everything exciting
+that's happened in Trino since Trino Fest, as well as a sneak peek at the
+roadmap for features coming to Trino in 2024. It's one of the best ways to keep
+up with the ongoing developments in the Trino community, and you won't want to
+miss it.
+
+A second talk from Starburst will be exploring the new Trino Gateway, a proxy
```

Comment:
This talk should be mentioning Bloomberg, Starburst and Naver at least .. maybe also Lyft